config.md

```diff
@@ -190,11 +196,27 @@ _Note: symbolic name for uid and gid, such as uname and gname respectively, are
     "apparmorProfile": "acme_secure_profile",
     "selinuxLabel": "system_u:system_r:svirt_lxc_net_t:s0:c124,c675",
     "noNewPrivileges": true,
-    "capabilities": [
-        "CAP_AUDIT_WRITE",
-        "CAP_KILL",
-        "CAP_NET_BIND_SERVICE"
-    ],
+    "capabilities": {
+        "bounding": [
+            "CAP_AUDIT_WRITE",
+            "CAP_KILL",
```

Comment:
the example should have `CAP_NET_BIND_SERVICE` in the bounding set or it cannot be applied as an ambient capability.